### PR TITLE
Pt 153897559 add transaction TTL

### PR DESCRIPTION
**Pull request by hanssv**

### CODE REVIEW

File: config/swagger.yaml [hunk 1]
```diff
@@ -2368,6 +2387,9 @@ definitions:
       fee:
         type: integer
         format: int64
+      ttl:
+        type: integer
+        format: int64
```
Comment (lucafavatella) [inline config/swagger.yaml]: This may impact (broken) code generators from swagger spec, though out Python client is the only one I am aware of.

Comment (hanssv) [reply config/swagger.yaml]: I think the swagger file would benefit from a normalization, the "same" things are expressed in a multitude of ways today.

**Comment by hanssv** [comment]
https://github.com/aeternity/protocol/pull/107

**Comment by lucafavatella** [comment]
Will need rebase.

**Comment by hanssv** [comment]
Rebased

**Comment by lucafavatella** [comment]
Was in scope of https://www.pivotaltracker.com/story/show/153897559

File: config/swagger.yaml [hunk 2]
```diff
@@ -2210,6 +2210,9 @@ definitions:
       fee:
         type: integer
         format: int64
+      ttl:
```
Comment (lucafavatella) [inline config/swagger.yaml]: Not added to required.

Comment (lucafavatella) [reply config/swagger.yaml]: Addressed in https://github.com/aeternity/epoch/pull/1217

File: config/swagger.yaml [hunk 3]
```diff
@@ -3005,6 +3044,10 @@ definitions:
         description: Transaction fee
         type: integer
         minimum: 0
+      ttl:
```
Comment (lucafavatella) [inline config/swagger.yaml]: Missing in require as noticed by @hanssv

Comment (lucafavatella) [reply config/swagger.yaml]: Addressed in https://github.com/aeternity/epoch/pull/1217

**Pull request merged**